package.json

```diff
@@ -142,6 +143,7 @@
     "css-loader": "^4.2.1",
     "csv-stringify": "^5.5.1",
     "dom-helpers": "^5.1.4",
+    "dotenv-webpack": "^7.0.2",
```

Comment:
Dependency added as a workaround to fix Storybook support for webpack 5 https://github.com/storybookjs/storybook/issues/14497#issuecomment-818723376